airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/job.py

```diff
@@ -302,6 +312,18 @@ def on_access_denied_job(self, **kwagrs) -> AirbyteTracedException:
     def on_job_with_errors(self, errors: List[Mapping[str, Any]]) -> AirbyteTracedException:
         raise ShopifyBulkExceptions.BulkJobUnknownError(f"Could not validate the status of the BULK Job `{self.job_id}`. Errors: {errors}.")
 
+    def on_retryable_error(self, response: requests.Response, exception: Exception) -> Optional[requests.Response]:
+        if self._job_retry_on_error_count == self._job_retry_on_error_max_limit:
+            self.on_job_with_errors(self.job_check_for_errors(response))
+        else:
+            # increment the attempt
+            self._job_retry_on_error_count += 1
+            # retry the request
```

Comment:
Maybe using more vertical spacing would help with readability. I believe the logging entry serves as an explanation of what is about to happen. And the retry_request method is clear. I suggest replacing this comment by a empty line.